gulpfile.js

```diff
@@ -160,6 +162,8 @@ async function createDist() {
 
 const recompile = gulp.series(
     clean,
+    async () => nugetInstall('CAP_ISVExp_Tools_Daily', 'Microsoft.PowerApps.CLI.Core.linux-x64', '1.5.6-daily-21031515', path.resolve(outdir, 'pac_linux')),
+    async () => { if (os.platform() === 'linux') { return fs.chmod(path.resolve(outdir, 'pac_linux', 'tools', 'pac'), 0o711) } },
```

Comment:
nit: the os check shouldn't be necessary: node's fs.chmod() does "succeed" on windows as well

Reply:
Nice.  Updated.